src/BizHawk.Emulation.Cores/Consoles/Nintendo/Gameboy/Gambatte.cs

```diff
@@ -101,7 +101,27 @@ public Gameboy(CoreComm comm, GameInfo game, byte[] file, Gameboy.GambatteSettin
 
 				if (_syncSettings.EnableBIOS)
 				{
-					bios = comm.CoreFileProvider.GetFirmware(biosSystemId, biosId, true, "BIOS Not Found, Cannot Load.  Change SyncSettings to run without BIOS.");
+					try
+					{
+						bios = comm.CoreFileProvider.GetFirmware(biosSystemId, biosId, true, "BIOS Not Found, Cannot Load.  Change SyncSettings to run without BIOS.");
+					}
+					catch (MissingFirmwareException)
+					{
+						if (_syncSettings.ConsoleMode != GambatteSyncSettings.ConsoleModeType.GBA)
+						{
+							throw;
+						}
+						else
+						{
+							//implicitly throws MissingFirmwareException if GBC BIOS is also missing
+							bios = comm.CoreFileProvider.GetFirmware(biosSystemId, "World", true, "BIOS Not Found, Cannot Load.  Change SyncSettings to run without BIOS.");
+							byte[] agbOverride = { 0xFF, 0x00, 0xCD, 0x03, 0x35, 0xAA, 0x31, 0x90, 0x94, 0x00, 0x00, 0x00, 0x00 };

```

Comment:
Would it make more sense to override to the real bios and not the gambatte one?

Reply:
Well, there are some issues for that.
1) Copyright (which yes you could argue fair use but eh, RE has a way better argument for that)
2) Trade secret laws (our knowledge of the code from the agb-cgb bios literally comes from illegal leaks)
3) Overall would make the code more complicated (since there's another change elsewhere in the BIOS... which doesn't change cycle counts and its only practical effects would be hardware only).

The Gambatte one is a bit more simple and comes from RE work so it would be far more legal to use than to deal with putting leak content in the fallback.

Reply:
Those sound like nasty consequences for either choice.  Maybe it's better to just not do this.